doc/news/changes/minor/20170927JuliusWitte

```diff
@@ -0,0 +1,10 @@
+Changed: Split TensorProductMatrixSymmetricSum into base and derived class.
+Added a template specialization for TensorProductMatrixSymmetricSum
+in case of having a VectorizedArray as arithmetic type.
+In addition, TensorProductMatrixSymmetricSum is now able to handle
+distinct 1D matrices in each tensor dimension. Moreover, the matrix-vector
+product methods, namely TensorProductMatrixSymmetricSumBase::vmult() and
+TensorProductMatrixSymmetricSumBase::apply_inverse(), now work on
+ArrayView instead of raw pointers.
```

Comment:
Since we have not yet released deal.II with `TensorProductSymmetricSum`, we do not need to mention the interface with `ArrayView` -> this makes the message more compact.